Makefile

```diff
@@ -32,12 +32,17 @@ docclean:
 livehtml:
 	$(MAKE) -C doc $@
 
+# The first dist target here exports a variable to the environment that keeps tar
+# from including the resource files on macOS.
+dist:   export COPYFILE_DISABLE = true
```

Comment:
(Wow, that's an odd features; doesn't even seem documented for their `tar`.)

Reply:
Apple's `tar` used to have an option called `--disable-copyfile` that you could pass, but they broke it in the last few versions of macOS.

Reply:
And yah, it's completely undocumented (and unfortunately means it's prone to later breakage). I pulled it out of https://superuser.com/a/260264.

Reply:
Yeah, that's fine, I was just expressing my amazement. :) Would suggest to put that link into a comment though.